### PR TITLE
Integration Tests through grade submission

### PR DESCRIPTION
**Pull request by muselesscreator**
Title says it all :-p

JIRA: [AU-456](https://openedx.atlassian.net/browse/AU-456)

### CODE REVIEW

File: src/test/app.test.jsx [hunk 1]
```diff
@@ -217,194 +199,276 @@ const waitForRequestStatus = (key, status) => waitForEqual(
...
+        await userEvent.click(await inspector.find.review.startGradingBtn());
+      });
+      /*
+        test('pending', async (done) => {
+          done();
+        });
+        test('error', async (done) => {
+          done();
+        });
+      */
```
Comment (leangseu-edx) [inline src/test/app.test.jsx:407]:
```suggestion
```

File: src/test/app.test.jsx [hunk 2]
```diff
@@ -217,194 +199,276 @@ const waitForRequestStatus = (key, status) => waitForEqual(
...
+          expect(current.lockStatus).toEqual(lockStatuses.unlocked);
+        }
+        /*
+          test('submit pending', async (done) => {
+            done();
+          });
+          test('submit failed', async (done) => {
+            done();
+          });
+        */
```
Comment (leangseu-edx) [inline src/test/app.test.jsx:461]:
```suggestion
```

**Review by leangseu-edx — APPROVED** [review]
Just remove a couple debris, the rest look good.

File: src/test/app.test.jsx (hunk 2 above)
Comment (muselesscreator) [reply src/test/app.test.jsx:461]: ? What is the change here?

Comment (leangseu-edx) [reply src/test/app.test.jsx:461]: removing the comment. I thought it was unintended.

Comment (muselesscreator) [reply src/test/app.test.jsx:461]: oh! no, is intentional placeholder because we need those tests next

**Pull request merged**